xbmc/network/Network.h

```diff
@@ -145,10 +170,65 @@ class CNetwork
    void StartServices();
    void StopServices(bool bWait);
 
+   // tests if string is VALID IPv6 or VALID IPv4 address
```

Comment:
@Paxxi 

sure, I can. do we have short howto somewhere? 
or looking at other comments, is it this ?
```
   /*!
    \brief  Translates a 802.11a+g frequency into the corresponding channel
    \param  frequency  The frequency of the channel in units of Hz
    \return The channel as an integer between 1 and 14 (802.11b+g) or
            between 36 and 165 (802.11a), or 0 if unknown.
    */
```